### PR TITLE
Add subctl cmd show gateways

### PR DESCRIPTION
**Pull request by maayanf24**
Resolve issue #468

Signed-off-by: Maayan Friedman <maafried@redhat.com>

### CODE REVIEW

File: pkg/subctl/cmd/show_gateways.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+			}
+		}
+
+		var summary string
+		if totalConnections > 0 && totalConnections == countConnected {
+			summary = "All connections OK"
+		} else {
+			summary = gateway.Status.StatusFailure
+		}
+		connectionString := fmt.Sprintf("%d/%d", countConnected, totalConnections)
```
Comment (tpantelis) [inline pkg/subctl/cmd/show_gateways.go:75]: `Status.StatusFailure`  (currently) is populated when there is a failure obtaining the connections in the driver. I think the connection string and the summary somewhat overlap plus if there's partial connectivity then the summary would be empty. I propose we simplify and combine the connection string with the summary:

```suggestion
                if gateway.Status.StatusFailure != "" {
                    summary = gateway.Status.StatusFailure
                } else if totalConnections == 0 {
                    summary = "There are no connections"
                } else if totalConnections == countConnected {
			summary = fmt.Sprintf("All %d connections are established",  totalConnections)
		} else {
			summary = fmt.Sprintf("%d connections out of %d are established",  totalConnections, countConnected)
		}
	
```

File: pkg/subctl/cmd/show_gateways.go [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+func showGateways(cmd *cobra.Command, args []string) {
+	var status []gatewaysStatus
+	status = getGatewaysStatus(status)
+	printGateways(status)
+}
+
+func printGateways(gateways []gatewaysStatus) {
+	template := "%-20s%-16s%-16s%-32s\n"
+	fmt.Printf(template, "NODE", "HA-STATUS", "CONNSCTIONS", "SUMMARY")
```
Comment (tpantelis) [inline pkg/subctl/cmd/show_gateways.go:90]:
```suggestion
	fmt.Printf(template, "NODE", "HA STATUS", "CONNECTIONS", "SUMMARY")
```

File: pkg/subctl/cmd/show_gateways.go (hunk 1 above)
Comment (maayanf24) [reply pkg/subctl/cmd/show_gateways.go:66]: in case there will be only 1 connection, the summary will show: `All 1 connections are established`

Comment (tpantelis) [reply pkg/subctl/cmd/show_gateways.go:66]: yeah good point. How about `All connections (%d) are established`?

Comment (maayanf24) [reply pkg/subctl/cmd/show_gateways.go:66]: sounds good.

Comment (maayanf24) [reply pkg/subctl/cmd/show_gateways.go:66]: shell I keep the `CONNECTIONS` column? 

```
[maafried@localhost submariner-operator]$ ./bin/subctl show gateways
NODE                HA STATUS       CONNECTIONS     SUMMARY                         
cluster2-worker     active          1/1             All connections (1) are established
```

Comment (tpantelis) [reply pkg/subctl/cmd/show_gateways.go:66]: no I think it's redundant.

**Pull request merged**